cmd/candidsrv/main.go

```diff
@@ -178,7 +186,6 @@ func serveIdentity(conf *config.Config, params candid.ServerParams) error {
 		Handler:   server,
 		TLSConfig: conf.TLSConfig(),
 	}
-	fmt.Println("START")
```

Comment:
I've just remembered what this is for. We use the START message to detect that the service is working. Please put it back.

Reply:
done